docs/GettingStarted.md

```diff
@@ -595,3 +595,6 @@ development cycle.  See the [Windows Hardware Developer documentation](https://l
 ## eBPF external extensions
 Extensions that integrate with eBPF for Windows:
 - XDP extension: [xdp-for-windows](https://github.com/microsoft/xdp-for-windows)
+- NTOS eBPF extensions:
+   - ntosebpfext extension: [ntosebpfext](https://github.com/microsoft/ntosebpfext)
+   - Network event extension: [neteventebpfext](https://github.com/microsoft/ntosebpfext/blob/main/docs/neteventebpfext.md)
```

Comment:
```suggestion
- ntosebpfext extension: [ntosebpfext](https://github.com/microsoft/ntosebpfext)
- Network event extension: [neteventebpfext](https://github.com/microsoft/ntosebpfext/blob/main/docs/neteventebpfext.md)
```
As I mentioned in my previous comment, I don't think "network event extension" is really an "NTOS" extension (or calling it such is confusing in any case)